### ISSUE TITLE
Worse translation results after updating to pytorch  0.4.0

### ISSUE THREAD

**Issue by HsinYingLee** [NVlabs/MUNIT#27]
Hi all,
After updating to pytorch 0.4.0, I try to train MUNIT on the summer2winter_yosemite dataset from Cyclegan (https://github.com/junyanz/CycleGAN) yet the results are of ill quality. I tried the initial released version of MUNIT before and it worked perfectly well. Is anyone facing the same issue after switching to pytorch 0.4.0?

Here is the snapshot after 150000 iterations:
![image](https://user-images.githubusercontent.com/14243520/43025387-69359656-8c26-11e8-99d9-4c4578f235a5.png)

I use the default configuration.

Thank you.

**Comment by MilanKolkata** [comment]
@HsinYingLee I also have the similar problem, how many images do you have in your training set(trainA and trainB)?

**Comment by MilanKolkata** [comment]
@HsinYingLee Thanks for mentioning it. I am trying the new code. How many images did you use when training the model?

**Comment by mingyuliutw** [comment]
@MilanKolkata In the commit [972e42](https://github.com/NVlabs/MUNIT/commit/f972e4237e2a8615c80950f2b987924256586e5c), the custom layernorm only supports one image per batch. With the new commit [4c21350](https://github.com/NVlabs/MUNIT/commit/4c21350603d83406a9712f1ea02aa5f564eea0ad), it supports multiple images per batch. However, the time required for each iteration would increase about 4 times when you use a batch size greater than 1. (This is due to the change of the way pytorch implements `view` function from 0.3 to 0.4. For training with batch size greater than 1, please roll back to pytorch 0.3 and use  [munit_pytorch0.3](https://github.com/NVlabs/MUNIT/releases/tag/munit_pytorch0.3). 

BTW, I am still confirming if the performance is the same for pytorch 0.3 and 0.4.

**Comment by mingyuliutw** [comment]
@HsinYingLee and @MilanKolkata I think the code is now working properly in pytorch 0.4 now. 

We spent sometime in the past few days playing with this `summer2winter_yosemite256`. We found that when enabling explicit cycle consistency loss, the model converges faster and better. The new config file can be found in `configs/summer2winter_yosemite256_folder.yaml` .
